.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
+omit = tests/*, scripts/*, docs/*
```

Comment:
Thanks for adding this. We don't have a `scripts ` folder here so could you remove that part?